### PR TITLE
Bump version to 3.5.0

### PR DESCRIPTION
**Pull request by wagewarbler**
Bump version to 3.5.0.

cc @HarelM

### CODE REVIEW

**Comment by HarelM** [comment]
Can you update the lock file as well please?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
 ### 🐞 Bug fixes
 
 - _...Add new stuff here..._
 
+## 3.5.0
+
+### ✨ Features and improvements
+
+- Add setTiles method to RasterTileSource to dynamically update existing tile
+  sources. ([3208](https://github.com/maplibre/maplibre-gl-js/pull/3208))
```
Comment (HarelM) [inline CHANGELOG.md:16]: This should be a single line, shouldn't it?

File: package-lock.json [hunk 2]
```diff
@@ -1939,97 +1939,6 @@
       "integrity": "sha512-VeiPZ9MMwXjO32/Xu7+OwflfmeoRwkE/qzndw42gGtgJwZopBnzy2gD//NN1+go1mADzkDcqf/KnFRSjTJ8xJA==",
       "dev": true
     },
-    "node_modules/@rollup/rollup-android-arm-eabi": {
```
Comment (HarelM) [inline package-lock.json:1942]: Something in this lock file isn't right... There should be minimal changes...

Comment (neodescis) [reply package-lock.json:1942]: Weird. Most of the removals are @rollup packages, but there's fsevents removed as well.

Comment (wagewarbler) [reply package-lock.json]: Hmm, strange. I've just reverted back to the package.json prior to the last commit and run an `npm install` against that one and pushed up the changes. Hopefully that'll do it. Sorry for the trouble!

**Review by neodescis — APPROVED** [review]

File: package-lock.json (hunk 2 above)
Comment (neodescis) [reply package-lock.json]: That seemed to do the trick. Looks good to me now!

**Pull request merged**